test/unit/processing/components/get-schedule-dates.test.js

```diff
@@ -95,79 +95,63 @@ describe('get schedule dates', () => {
 
   test('should insert top up adjustment after last outstanding payment in original schedule', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[2].period).toEqual(moment().format('MMM YYYY'))
+    expect(schedule[0].period).toEqual(moment().format('MMM YYYY'))
   })
 
   test('Adjustment payment Type must be Immediate payment', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[2].paymentType).toEqual(IMMEDIATE)
-  })
-
-  test('should retain previous payment value for paid segments if top up', () => {
```

Comment:
Can we instead of removing these tests, modify them so we're checking that previous schedules aren't outputted. Helps with regression testing.